src/storage-blob-preview/azext_storage_blob_preview/_help.py

```diff
@@ -6,6 +6,29 @@
 
 from knack.help_files import helps  # pylint: disable=unused-import
 
+helps['storage account blob-service-properties update'] = """
```

Comment:
This change has nothing to do with jumbo blob, it's aimed to fix CI.

As in cli main repo, a new argument(`--default-service-version`) is added to this command and the help example added the usage of this new argument. But this command is overwritten in extension to apply track2 sdk and no new argument added, so if we keep using help from main repo, CI will fail because in extension CI it can't recognize the new argument.

Reply:
As extension is preview for storage module, we should make it newer than main repo.
For the feature added in main repo, we should support it in this extension, too.
Could you add `--default-service-version` in extension command, just copy my code in main repo?

Reply:
Sure